### PR TITLE
Feature/#143 nav concordia menu

### PR DESCRIPTION
**Pull request by marwan011**
- add a menu that contains events and settings 
- add a picker to display today's events of the campus selected

### CODE REVIEW

File: components/Menu.tsx [hunk 1]
```diff
@@ -0,0 +1,296 @@
...
+  }
+  closeZeMenu() {
+    this.setState({ showMenu: false });
+  }
+  closeZeEvents() {
+    this.setState({ showEvent: false });
+  }
+  closeZeSettings() {
+    this.setState({ showSettings: false });
+  }
```
Comment (Alessandro100) [inline components/Menu.tsx:167]: Whats with the 'Ze' naming convention?

File: components/Menu.tsx [hunk 2]
```diff
@@ -0,0 +1,296 @@
...
+> {
+  constructor(props) {
+    super(props);
+
+    this.state = {
+      showMenu: false,
+      showEvent: false,
+      showSettings: false,
+      buildings: obtainBuildings(),
+      building: new Building("", "", null, null, "", null, "H"),
```
Comment (Alessandro100) [inline components/Menu.tsx:152]: If you are initializing a null building, make sure the ending is also 'null' and not 'H'

Comment (marwan011) [reply components/Menu.tsx:152]: resolved

File: components/Menu.tsx (hunk 1 above)
Comment (marwan011) [reply components/Menu.tsx:167]: resolved

File: components/Menu.tsx (hunk 2 above)
Comment (marwan011) [reply components/Menu.tsx:152]: resolved

File: components/Menu.tsx (hunk 1 above)
Comment (marwan011) [reply components/Menu.tsx:167]: resolced

File: components/Menu.tsx (hunk 2 above)
Comment (marwan011) [reply components/Menu.tsx:33]: I will let this one to H since the first building in the picker is H. So by default it will show the first picker.

**Comment by Alessandro100** [comment]
Hey great changes! I like the use of your comments as well as seperating the styling in the menu component. One thing is, the commit you made for the changes didn't link the issue correctly. The number needs to be connected to the # (#143), you added a space (# 143). This wont link the issue. If you can just change that and add somewhere in the name that this was a PR fix, Ill accept it

**Pull request merged**